### DISCUSSION TITLE
Controlling view orientation

### DISCUSSION THREAD

**Discussion by josmitty** [Stellarium/stellarium#3397] [Q&A] [answered]
When launching Stellarium, the "camera" is pointing toward South.  Clicking with left mouse button, I can drag the "camera" to change its orientation (yaw and pitch), i.e. to look more toward East or West or to look up or down.   That works well, but it is done with the mouse and so is not precise.

Is there a way to enter orientation numbers in the UI to precisely select the orientation?  Or maybe a way to display the orientation angles in real-time when moving with the mouse to see what orientation we are precisely at?

Thanks.

**Comment by gzotti** [comment, marked as answer]
It helps to read the User Guide all the way to page 15 or so, or even further.
Apart from intuitive mouse and less intuitive keyboard control and some hotkeys for viewing towards the cardinals, there is also scripting, and a mouse coordinate plugin.
